--- before
+++ after
@@ -1,12 +1,12 @@
 Building configuration...
 
-Current configuration : 5556 bytes
+Current configuration : 5578 bytes
 !
 ! No configuration change since last restart
 !
 version 12.4
 service timestamps debug datetime msec
-service timestamps log datetime localtime
+service timestamps log datetime msec
 no service password-encryption
 no service dhcp
 !
@@ -158,6 +158,7 @@
 no ip http secure-server
 !
 logging trap notifications
+logging origin-id hostname
 logging host 192.168.122.1 transport udp port 10514
 snmp-server community sentryPod RO
 snmp-server enable traps snmp authentication linkdown linkup coldstart warmstart

