--- before
+++ after
@@ -1,10 +1,12 @@
 Building configuration...
 
-Current configuration : 5857 bytes
+Current configuration : 5951 bytes
+!
+! Last configuration change at 16:58:48 SLST Wed Jun 10 2026 by admin
 !
 version 15.2
 service timestamps debug datetime msec
-service timestamps log datetime localtime
+service timestamps log datetime msec
 !
 hostname R2
 !
@@ -100,6 +102,7 @@
 ip route 192.168.8.101 255.255.255.255 FastEthernet1/0
 !
 logging trap notifications
+logging origin-id hostname
 logging host 192.168.122.1 transport udp port 10514
 !
 snmp-server community sentryPod RO

