--- before
+++ after
@@ -1,10 +1,10 @@
 Building configuration...
 
-Current configuration : 5802 bytes
+Current configuration : 5824 bytes
 !
 version 15.2
 service timestamps debug datetime msec
-service timestamps log datetime localtime
+service timestamps log datetime msec
 !
 hostname R1
 !
@@ -100,6 +100,7 @@
 no ip http secure-server
 !
 logging trap notifications
+logging origin-id hostname
 logging host 192.168.122.1 transport udp port 10514
 !
 snmp-server community sentryPod RO

